python/lsst/faro/measurement/TractTableMeasurementTasks.py

```diff
@@ -21,17 +21,57 @@
 
 from lsst.pex.config import Config, Field
 from lsst.pipe.base import Struct, Task
+from lsst.pipe.tasks.configurableActions import ConfigurableActionStructField
 from lsst.verify import Measurement, Datum
 
+from lsst.faro.base.ConfigBase import MeasurementTaskConfig, ColumnField, ColumnBandField
+import lsst.faro.utils.selectors as selectors
 from lsst.faro.utils.tex_table import calculateTEx
 
 import astropy.units as u
 import numpy as np
 
-__all__ = ("TExTableConfig", "TExTableTask")
+__all__ = ("TExTableConfig", "TExTableTask", "NumSourcesTask") #, "NumSourcesConfig"
```

Comment:
Clean up commented part if not needed.

Reply:
Will fix once we decide how to move forward with:
https://github.com/lsst/faro/pull/117#discussion_r791870903

Reply:
We have decided to fully remove "NumSourcesTask" from python/lsst/far/measurement/TractTableMeasurementTasks.py